src/overlays/actors/ovl_En_Md/z_en_md.c

```diff
@@ -15,11 +15,11 @@ void EnMd_Destroy(Actor* thisx, PlayState* play);
 void EnMd_Update(Actor* thisx, PlayState* play);
 void EnMd_Draw(Actor* thisx, PlayState* play);
 
-void func_80AAB874(EnMd* this, PlayState* play);
-void func_80AAB8F8(EnMd* this, PlayState* play);
-void func_80AAB948(EnMd* this, PlayState* play);
-void func_80AABC10(EnMd* this, PlayState* play);
-void func_80AABD0C(EnMd* this, PlayState* play);
+void EnMd_ActionIdle(EnMd* this, PlayState* play);
+void EnMd_ActionWatch(EnMd* this, PlayState* play);
+void EnMd_ActionBlockPath(EnMd* this, PlayState* play);
+void EnMd_ActionListenToOcarina(EnMd* this, PlayState* play);
+void EnMd_ActionWalk(EnMd* this, PlayState* play);
```

Comment:
Player has it because of its complexity and having multiple types of "action funcs"
Personally dont think we need to make a bunch of names longer, when in most actors there is only one action func and it is the only thing handling all of its functionality

Anyway, agree on separating this discussion from the PR